lib/MetricsProducer.js

```diff
@@ -56,7 +56,7 @@ class MetricsProducer {
                     // Using trace here because errors are already logged in
                     // BackbeatProducer. This is to log to see source of caller
                     this._log.trace(`error publishing ${type} metrics for` +
-                        `extension metrics ${ext}`);
+                        `extension metrics ${ext}`, err);
```

Comment:
it's a trace log so does not matter much, but the format should be `{ error: err.message }`